build.gradle

```diff
@@ -4,10 +4,11 @@ plugins {
   id 'pmd'
   id 'jacoco'
   id 'io.spring.dependency-management' version '1.1.4'
-  id 'org.springframework.boot' version '3.2.0'
+  id 'org.springframework.boot' version '3.1.5'
```

Comment:
What's the reason for the downgrade?

Reply:
Version 3.2.0  breaks the enum mapping code. Meaning you cannot start the app due to this error:
```
2023-11-30T13:50:20.736 ERROR [main] org.springframework.boot.SpringApplicationorg.springframework.beans.factory.BeanCreationException: Error creating bean with name 'entityManagerFactory' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/HibernateJpaConfiguration.class]: org/hibernate/type/descriptor/converter/spi/EnumValueConverter
```

I found this issue about it- seems like there will be a fix soon:  https://github.com/spring-projects/spring-framework/issues/31705

Reply:
can u add a comment to that effect next to the code please else we may get someone accidentally trying to updated - ta